docs/proposals/Disclosure/README.md

```diff
@@ -0,0 +1,111 @@
+# Disclosure
+
+A `<Disclosure>` component will reveal and hide additional information that our
+users may care about, should they choose to learn more about it.
+
+## Design Usage and Guidelines
+
+- use when you want to reduce feelings of being overwhelmed and to reduce
+  distractions
+- used for lightweight editing, such as selecting a value for a setting
```

Comment:
I'm a little fuzzy on what we mean by this line - are we trying to convey something like "can be used to hide optional settings controls"?

Reply:
I agree, that sounds much better